corehq/apps/data_dictionary/templates/data_dictionary/partials/confirmation_modals.html

```diff
@@ -117,14 +117,12 @@ <h4 class="modal-title">
             <div class="modal-body">
                 <p>
                     {% blocktrans %}
-                        There are currently no cases that store data with this case property, so it is safe to delete.
+                        There are currently <span id="delete-case-prop-count"></span> that store data with this case property.
```

Comment:
Idle thought: What if you just didn't show a message if there are no cases using the case property? Do you think that would make the modal better, or would it be obscuring info that's valuable to users?